docs/RFCS/monitoring_queries_and_sessions.md

```diff
@@ -0,0 +1,105 @@
+- Feature Name: Monitoring Queries and Sessions
+- Status: in-progress
+- Start Date: 2017-05-04
+- Authors: Bilal Akhtar
+- RFC PR: https://github.com/cockroachdb/cockroach/pull/15761
+- Cockroach Issue: [#7003](https://github.com/cockroachdb/cockroach/issues/7003)
+
+# Summary
+
+This feature would add a mechanism to list currently active sessions and currently executing queries
+in the admin UI, along with their start timestamps  / durations
+
+# Motivation
+
+Currently, there's no visibility into what queries/sessions are running on the cluster at any
+given point of time. Adding visibility into this has been a common customer request for
+a while, along with the ability to terminate long-running queries and specific sessions.
+
+# Detailed design
+
+The inspiration behind some of the design comes from
+[this PR from Raphael](https://github.com/cockroachdb/cockroach/pull/10317) which implemented
+node-local session registries exposed via a virtual table.
+
+## New unique UUID identifiers for sessions and queries
```

Comment:
Making these two changes:

- Explicitly stating that node IDs can be prefixed to this UUID when presented, separated by a delimeter. The purpose of the UUID is only to identify it locally within a node.

- Adding the pointer hash alternative to the Alternatives section.